main.go

```diff
@@ -75,11 +66,18 @@ func cmdAdd(args *skel.CmdArgs) error {
 		},
 	}
 
+	metadataRoute, err := cniConf.getMetadataRoute()
+	if err != nil {
+		logrus.Errorf("rancher-flat-ipam: error getting metadata route :%v", err)
+		return err
+	}
+
 	r.IP4.Routes = append(
 		ipamConf.Routes,
```

Comment:
Can you please append metadataRoute, followed by ipamConf.Routes?

Reply:
```
ipamConf.Routes = append(ipamConf.Routes, metadataRoute)
r.IP4.Routes = append(ipamConf.Routes,)
```
or
```
r.IP4.Routes = append(
      ipamConf.Routes,
      metadataRoute,
)
```
What's your point?
That doesn't seem to make any difference.

Reply:
r.IP4.Routes = append(
      metadataRoute,
      ipamConf.Routes,
)